### ISSUE TITLE
Decorative SVG elements should be hidden using aria-hidden="true"

### ISSUE THREAD

**Issue by danchamp** [localgovdrupal/localgov_base#111]
[Accessibility audit: issue 57](https://docs.google.com/spreadsheets/d/1t8rRIQe_1kogxPBZDKcsbf8YxtEfC9iGJvidkagNSXY/edit#gid=1779042480&range=E58)

**Comment by danchamp** [comment]
Closing, no action required:

[icon.html.twig](https://github.com/localgovdrupal/localgov_base/blob/ceb0f6741beadacc433f5367706fadbd3a0d8b8b/templates/includes/icons/icon.html.twig) sets `aria-hidden` on the icon wrapper element, and provides for passing a `decoration` parameter to control it - nice work @markconroy!
